src/modules/FavoritesPage/FavoritesPage.tsx

```diff
@@ -1,7 +1,70 @@
-import './FavoritesPage.module.scss';
-
 import React from 'react';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import styles from './FavoritesPage.module.scss';
+import homeIcon from './icons/Home.svg';
+import homeIconDark from './icons/HomeDark.svg';
+import arrowIcon from './icons/Chevron (Arrow Right).svg';
 
 export const FavoritesPage: React.FC = () => {
-  return <h1>Favorites Page</h1>;
+  const favourites: number[] = [1, 2, 3, 4, 5];
+  const isThemeDark = false;
+
+  return (
+    <section
+      className={cn(styles.favouritesPage,
+        { [styles.favouritesPageDark]: isThemeDark })}
+    >
+      <div className={styles.favouritesPage__top}>
+        <Link
+          to="/"
+        >
+          {isThemeDark
+            ? (
+              <img
+                src={homeIconDark}
+                alt="home-icon"
+              />
+            ) : (
+              <img
+                src={homeIcon}
```

Comment:
svg can be made a a react component and you just can changed fill color of your svg, can be really useful here)